agent/engine/docker_task_engine.go

```diff
@@ -313,10 +313,10 @@ func (engine *DockerTaskEngine) monitorExecAgentRunning(ctx context.Context,
 	status, err := engine.execCmdMgr.RestartAgentIfStopped(ctx, engine.client, task, c, dockerID)
 	if err != nil {
 		seelog.Errorf("Task engine [%s]: Failed to restart ExecCommandAgent Process for container [%s]: %v", task.Arn, dockerID, err)
-		mTask.emitManagedAgentEvent(mTask.Task, c)
+		mTask.emitManagedAgentEvent(mTask.Task, c, "Task engine unable to restart the Execute Command Agent")
```

Comment:
Should we send the actual `error.Error()` here?

Reply:
This shows up in the reason field for describe-tasks -- would be nice to keep it terse.
aka
"Exec Command Agent Restarted"